src/components/heroBanner/HeroBanner.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import {
+  LeftStyledHeroBanner,
+  CenterStyledHeroBanner,
+} from "./HeroBanner.styled";
+import { Button } from "../button/Button";
+import { Link } from "react-router";
+
+export const HeroBannerVariant = {
+  left: LeftStyledHeroBanner,
+  center: CenterStyledHeroBanner,
+};
+
+export const HeroBanner = ({ variant, img, alt, h2, p }) => {
+  const Tag = variant;
+  return (
+    <div className="container">
+      <Tag>
+        <img src={img} alt={alt} />
+        <div className="text">
+          <h2>{h2}</h2>
+          <p>{p}</p>
```

Comment:
if you inspect the h2 and p tag you'll find that they have innate top and bottom margin. On figma, there should only be a 12px gap between h2 and p, and 20px gap between p and button, so please fix :)